<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SttReplayParaSetWidget</class>
 <widget class="QWidget" name="SttReplayParaSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>409</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSttTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>采样率</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="1">
        <widget class="QTableWidget" name="tableWidgetIns">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>瞬时值</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>采样率</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>采样数</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSttTabWidget</class>
   <extends>QTabWidget</extends>
   <header>../Controls/SttTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
